docs/reference/anitya.lib.backends.rst

```diff
@@ -0,0 +1,12 @@
+Anitya Backends
+===============
+
+These are the backends supported by Anitya
+
+Crates.io
+---------
+
+.. automodule:: anitya.lib.backends.crates
```

Comment:
I like that idea but I think I would favor having it in a different PR so that all the modules can be added not just this one